src/main/java/pages/LoginPage.java

```diff
@@ -0,0 +1,40 @@
+package pages;
+
+import enums.BrowserName;
+import org.openqa.selenium.By;
+
+public class LoginPage extends BasePage {
+
+    // Constructor:
+    public LoginPage(BrowserName browserName) {
+        super(browserName);
+    }
+
+    // Locators:
+    private By loginInputLocator = By.xpath("//*[@id=\"loginFrm_loginname\"]");
+    private By passwordInputLocator = By.xpath("//*[@id=\"loginFrm_password\"]");
+    private By loginButtonLocator = By.xpath("//*[@id=\"loginFrm\"]/fieldset/button");
```

Comment:
This locator might be unstable since it's assuming we have a strict structure in the code.
How about more loose one: `//*[@id=\"loginFrm\"]//button`?

Reply:
Also you can use single qoutes inside the selector: `By.xpath("//*[@id='loginFrm_password']")`

Reply:
@dariabajda After today's discussion, we agreed on simply choosing the best possible syntax. These locators have now been changed to By.id() / Changed double quoting to single quoting for the 'loginButtonLocator' (which is By.xpath)